src/posix/platform/openthread-posix-config.h

```diff
@@ -252,6 +252,16 @@
 #endif
 #endif
 
+/**
+ * @def OPENTHREAD_POSIX_CONFIG_THREAD_DEFAULT_NET_IF_NAME
+ *
+ * Define the Thread default network interface name.
+ *
+ */
+#ifndef OPENTHREAD_POSIX_CONFIG_THREAD_DEFAULT_NET_IF_NAME
+#define OPENTHREAD_POSIX_CONFIG_THREAD_DEFAULT_NET_IF_NAME "wpan0"
```

Comment:
nit: how about renaming to:
```suggestion
#define OPENTHREAD_POSIX_CONFIG_THREAD_NETIF_DEFAULT_NAME "wpan0"
```

Reply:
Updated